index.js

```diff
@@ -1 +1,14 @@
 // Code your solution here
+function findMatching(drivers, name) {
+  const lowercaseName = name.toLowerCase();
+  return drivers.filter(driver => driver.toLowerCase().includes(lowercaseName));}
+  
+function matchName(array, name) {
+  return array.filter(driver => driver.name === name);
+}
+
+function fuzzyMatch(array, name) {
+  return array.filter(function(driver) {
+  return driver.substring(0, 1) === name.substring(0, 1);
+  });
+}
```

Comment:
So this works, and passes the test, but only if the name provided is 2 letters long. Take a peak at the `startsWith` function. You don't need to make changes, but if you have extra time, play around with it. Thanks!
```suggestion
function fuzzyMatch(array, name) {
  return array.filter(function(driver) {
    return driver.substring(0, 1) === name.substring(0, 1);
  });
}
```